src/groundlight/client.py

```diff
@@ -214,6 +222,15 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments
         if human_review is not None:
             params["human_review"] = human_review
 
+        if want_async is True:
+            # If want_async is True, we don't want to wait for a result. As a result wait must be set to 0 to use
+            # want_async.
+            if wait != 0:
+                raise ValueError(
+                    "wait must be set to 0 to use want_async. Using wait and want_async at the same time is incompatible."  # noqa: E501
+                )
+            params["want_async"] = want_async
```

Comment:
I think here we should be more deliberate about what we send to the API.  We can be pretty flexible in what the SDK method uses to indicate that they want async - they can give us an actual bool, or a 0/1 int or something else which indicates truthiness.  But we should be more specific about what we send to the server.  In the other PR I argue for always sending “1” or “0”.

Reply:
Why not just True/False?

Reply:
I'd defer to Leo. My understanding is the the "1" "0" distinction fixes the bits representation. True values can have multiple representations (that occasionally are different by system) so we might as well choose a 1:1 mapping on the values sent into our system rather than a many to one mapping. 
Again, I defer to Leo

Reply:
I disagree with this pretty strongly as we are going to essentially going to have to write a two translation layers (on both the SDK and BE) to convert to and from these "string bools". OTOH, python's `True` and `False` are already unambiguous values for T/F. Additionally the SDK automatically handles serialization of `True`/`False` and our BE automatically handles deserialization to `True`/`False` so no custom translation is required.

Reply:
If we really want this, I think we should do it as its own PR as that way we can focus on applying this update uniformly as there is BE work required as well.

Reply:
If we want to accept other arguments to represent truthiness from the user I think the solution is to do validation on the inputs prior to whatever business logic we do.

Reply:
```suggestion
            params["want_async"] = str(bool(want_async))
```
is what I mean.  Don't rely on the variable's python-truthiness and its string representation being the same.